source/Octopus.Versioning.Tests/Octopus/OctopusVersionTests.cs

```diff
@@ -683,32 +683,6 @@ public void TryParseTest()
             "0",
             "alpha",
             "")]
-        [TestCase("1_0_alpha",
-            1,
-            0,
-            0,
-            0,
-            0,
-            0,
-            0,
-            "",
-            "0_alpha",
-            "0",
-            "alpha",
-            "")]
-        [TestCase("1_0alpha",
-            1,
-            0,
-            0,
-            0,
-            0,
-            0,
-            0,
-            "",
-            "0alpha",
-            "0alpha",
-            "",
-            "")]
```

Comment:
It looks like these tests are checking if the maven version parser and octopus version parser returns the same result for certain version strings, these test cases that I removed no longer return the same result between the two version parsers. 

That said I think the maven parser suffers the same issue where it doesn't require a delimiter between the major version and the rest of the version string

Reply:
It might be worth running this one by @mcasperson who was involved in adding the maven support